src/templates/Base/styles.ts

```diff
@@ -0,0 +1,57 @@
+import styled, { css } from 'styled-components'
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100vh;
+`
+
+export const SectionHeader = styled.section`
+  ${({ theme }) => css`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 6.4rem;
+    z-index: ${theme.layers.menu};
+
+    position: fixed;
+    top: 0;
+    left: auto;
+    right: 0;
+    width: 100%;
+
+    background-color: ${theme.colors.white};
+    border-bottom: 1px solid rgba(0, 0, 0, 0.12);
+  `}
+`
+
+export const Logo = styled.img`
+  width: 10rem;
+  height: 3rem;
+`
+
+export const Content = styled.main`
+  ${({ theme }) => css`
+    padding-top: calc(6.4rem + (${theme.grid.gutter} * 2));
+  `}
+`
+
+export const SectionFooter = styled.section`
```

Comment:
Se o nome é `Footer`, pq não usar `footer`?
```suggestion
export const SectionFooter = styled.footer`
```